pkg/build/api/types.go

```diff
@@ -937,6 +941,10 @@ type BuildRequest struct {
 	// Env contains additional environment variables you want to pass into a builder container.
 	Env []kapi.EnvVar
 
+	// Args contains any build arguments that are to be passed to Docker.
```

Comment:
This doesn't belong in this structure unless build args are going to be generic to all strategies.